shop.py

```diff
@@ -100,12 +100,13 @@ def category_to_pretty_string(item, valid):
 
 class SellableItem():
     '''A class to represent an item that can be sold in the shop. This is a wrapper around the actual item, and includes a price.'''
-    def __init__(self, item, price=None):
+    def __init__(self, item, price=None, discount=0.0):
         self.item = item
         if price is not None:
           self.price = price
         else:
           self.price = self.set_price()
```

Comment:
How is the `.set_price()` method going to work? Is it just a dictionary that outputs a random value between a range based on the item type or something else?

Reply:
Yeah. [The current implementation is here](https://github.com/Vesper-arch/Slay-The-Spire-in-Python/pull/57/files#diff-7055173ca299d261e48a84f0a7d8ddc4c6429cbd527edf880ec8a180f5388170R119-R132), but needs to account for relics and potions. I was going to hardcode [the rules in the wiki](https://slay-the-spire.fandom.com/wiki/Merchant), and then have some kind of discount/override mechanism for interactions with relics and other things.

Reply:
oh

Reply:
i didn't see that